.github/workflows/test.yml

```diff
@@ -15,10 +15,17 @@ jobs:
           dj30_cms37.txt,
           dj30_cms38.txt,
           dj31_cms38.txt,
+          dj32_cms38.txt,
+          dj32_cms39.txt,
+          # dj32_cms310.txt,
```

Comment:
This can come into play now 👍

Reply:
I forgot about this.... They are in play now!